### DISCUSSION TITLE
Bricked CPC200-U2W Plus

### DISCUSSION THREAD

**Discussion by heycode-eu** [ludwig-v/wireless-carplay-dongle-reverse-engineering#89] [Q&A]
Hello,

I've upgraded my CPC200-U2W Plus using its web GUI, but after that, solid RED Light is on, and it's not working anymore.

Can't see Wi-Fi, bluetooth, or anything on the screen.

I've tried to connect an USB stick with the upgrade, with all file names suggested in the guides, but nothing happens.

Do you have any idea to recover it?

**Comment by Nesto0o** [comment]
I have the exact same problem here, don’t know what to do….

**Comment by Tozzi89** [comment]
Same issue here, please advise.

**Reply by Tozzi89** [reply]
Solved my issue!

If you are using CarlinKit 3.0 with red UI and black carbon case: https://github.com/ludwig-v/wireless-carplay-dongle-reverse-engineering/discussions/96#discussioncomment-4737785

You can't upgrade it while it is conencted to a PC or the car, you must use a USB charger/adapter. Apparently the signals on the databus can interupt the upgrade and brick it.
